Misc/NEWS.d/next/Library/2024-02-11-07-31-43.gh-issue-82062.eeS6w7.rst

```diff
@@ -0,0 +1,3 @@
+Fix :func:`inspect.signature()` to correctly handle globals in extension
+modules.  Now signatures are correct if globals are use to define default
```

Comment:
```suggestion
modules.  Now signatures are correct if globals are used to define default
```

Reply:
Thanks, but I think @JelleZijlstra variant is better.